cookbook/typescript-agent/nodes.ts

```diff
@@ -0,0 +1,172 @@
+import { parse } from 'yaml'
+import { Node } from "../../typescript/brainyflow";
```

Comment:
```suggestion
import { Node } from "brainyflow";
```

we should make our code easy to copy/paste.
it's better to not use relative paths in import statements, and then use `npm link` to handle relative paths for us.

at the end, my local `package.json` contain a line similar to `"brainyflow": "link:../../typescript/brainyflow", which is a way to centralize the solution in a single line for the whole project.